docs/specification.md

```diff
@@ -940,14 +850,12 @@ Parameter (in Michelson):
 (list %burn
   (pair
     (address %sender)
-    (pair
-      (nat %tokenId)
-      (nat %value)
-  ))
+    (nat %value)
+  )
 )
 ```
 
-- Decreases balances for senders and their token types and the total supply of tokens by the given amount.
+- Decreases balances for senders and the total supply of tokens by the given amount.
```

Comment:
Might also mention that

> Burning tokens will not increase the mintingAllowance of the address doing the burning.

https://github.com/centrehq/centre-tokens/blob/78d964a1a8d481ffd8152772d7a66e47df54b3db/doc/tokendesign.md#burning

Reply:
Yes, will add.